PhysicsTools/PatAlgos/python/slimming/modifyPrimaryPhysicsObjects_cff.py

```diff
@@ -22,6 +24,7 @@
 
 modifyPrimaryPhysicsObjects = cms.Sequence( electronMVAValueMapProducer *
                                             photonIDValueMapProducer * photonMVAValueMapProducer * 
+                                            electronRegressionValueMapProducer * photonRegressionValueMapProducer *
```

Comment:
On 8/24/15 2:12 AM, Lindsey Gray wrote:
> In
> PhysicsTools/PatAlgos/python/slimming/modifyPrimaryPhysicsObjects_cff.py
> <https://github.com/cms-sw/cmssw/pull/10693#discussion_r37724172>:
> 
>> @@ -22,6 +24,7 @@
>>  
>>  modifyPrimaryPhysicsObjects = cms.Sequence( electronMVAValueMapProducer *
>>                                              photonIDValueMapProducer * photonMVAValueMapProducer * 
>> +                                            electronRegressionValueMapProducer * photonRegressionValueMapProducer *
> 
> This sequence isn't used in MiniAOD (but only in MiniAOD-from-MiniAOD,
> which was never fully setup in 75X/76X). That's why these differences
> are here.


So, do we need a follow-up PR in 75X/76X to have the
MiniAOD-from-MiniAOD working there?

The timescale of moving to 75X is not that long.


> … <#>
> On Mon, Aug 24, 2015 at 7:18 AM, Slava Krutelyov
> <notifications@github.com> wrote: In
> PhysicsTools/PatAlgos/python/slimming/modifyPrimaryPhysicsObjects_cff.py
> <https://github.com/cms-sw/cmssw/pull/10693#discussion_r37719977>: > @@
> -22,6 +24,7 @@ > > modifyPrimaryPhysicsObjects = cms.Sequence(
> electronMVAValueMapProducer * > photonIDValueMapProducer *
> photonMVAValueMapProducer * > + electronRegressionValueMapProducer *
> photonRegressionValueMapProducer * these are not in 75X/76X why needed
> here? Isn't the modification in egmGsfElectronIDSequence enough? — Reply
> to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10693/files#r37719977>.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10693/files#r37724172>.
>